lib/commons/builder/legislative_term.rb

```diff
@@ -0,0 +1,28 @@
+class LegislativeTerm
+  def initialize(legislature:, term_item_id: nil, start_date: nil, end_date: nil)
+    @legislature = legislature
+    @term_item_id = term_item_id
+    @start_date = start_date
+    @end_date = end_date
+  end
+
+  attr_accessor :legislature, :term_item_id, :start_date, :end_date
+
+  def query(language_map)
```

Comment:
I think this change to the method parameters is one that should be in the later commit.